elastic/apm-server
# Add transaction.experience metrics

## Motivation/summary

Add "experience" to the transaction schema, an
object which captures user/browser experience
metrics.

## Checklist


I have considered changes for:
~- [ ] documentation~
~- [ ] logging (add log lines, choose appropriate log selector, etc.)~
~- [ ] metrics and monitoring (create issue for Kibana team to add metrics to visualizations, e.g. [Kibana#44001](https://github.com/elastic/kibana/issues/44001))~
~- [ ] telemetry~

## How to test these changes

1. Start server
2. Send RUM transaction with experience metrics as described in #3972 (both v2 and v3)
3. Check the transaction docs for `transaction.experience` fields
4. Check that the fields are indexed

## Related issues

Closes #3972
